src/main/kotlin/app/revanced/patches/songpal/badge/fingerprints/ShowNotificationFingerprint.kt

```diff
@@ -30,7 +30,7 @@ object ShowNotificationFingerprint : MethodFingerprint(
 ) {
     val expectedReference = ImmutableMethodReference(
         "Lcom/google/android/material/bottomnavigation/BottomNavigationView;",
-        "showNotification", // Placeholder.
+        "f", // While the method is called "f" in 9.5.0, this may change in a future version, so it is not used for matching.
```

Comment:
I don't think "f" is useful, just give the method reference an explaining name such as `showNotification` and comment the version of the app this fingerprint was made for somewhere